libs/autofile/group_test.go

```diff
@@ -26,7 +26,7 @@ func createTestGroup(t *testing.T, headSizeLimit int64) *Group {
 	g, err := OpenGroup(headPath)
 	require.NoError(t, err, "Error opening Group")
 	g.SetHeadSizeLimit(headSizeLimit)
-	g.stopTicker()
+	g.ticker.Stop()
```

Comment:
isn't the previous version a bit more defensive?  we might want to add more operations there in the future, so at least the tests then wouldn't break.

Reply:
Fine :) I went with an easiest option. See my new commit https://github.com/tendermint/tendermint/pull/2135/commits/b1cff0f9bffd4d88ca2c6e9b9bc8e6d44cd3b463 where I moved the ticker to `OnStart`

```
create a Group ticker on Start

    1) no need to stop the ticker in createTestGroup() method
    2) now there is a symmetry - we start the ticker in OnStart(), we stop it
```